src/org/parosproxy/paros/extension/state/ExtensionState.java

```diff
@@ -52,7 +52,7 @@
 	private JCheckBoxMenuItem menuSessionTrackingEnable = null;
 
 	private ZapMenuItem menuResetSessionState = null;
-
+	

```

Comment:
If you could revert these formatting changes that'd be great.

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines